### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by NMinhNguyen**

### CODE REVIEW

File: pages/learn/basics/assets-metadata-css/index.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ In this lesson, you’ll learn:
 
 - How to add static files (images, etc) to Next.js.
 - How to customize what goes inside the `<head>` for each page.
-- How to create a reusable React.js component which is styled using CSS Modules.
+- How to create a reusable React component which is styled using CSS Modules.
```
Comment (NMinhNguyen) [inline pages/learn/basics/assets-metadata-css/index.mdx:22]: `React component` seems to be used a lot more often throughout the site as well as the rest of this tutorial, so I thought I'd drop the `.js` for consistency

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**…tes/api-routes-details**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/api-routes/api-routes-details) | [**…cs/assets-metadata-css**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/assets-metadata-css) |
|:-:|:-:|
|<a href="https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/api-routes/api-routes-details">Screenshot of /learn/basics/api-routes/api-routes-details</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/8nFPxFNoXSASm_BQuNLu9qPNL_o_.png">(view full size)</a> |<a href="https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/assets-metadata-css">Screenshot of /learn/basics/assets-metadata-css</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/_sn3uG1BHpIzgT0po9sEFHku1JA_.png">(view full size)</a> |



| [**…a-css/layout-component**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/assets-metadata-css/layout-component) | [**…-metadata-css/metadata**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/assets-metadata-css/metadata) |
|:-:|:-:|
|<a href="https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/assets-metadata-css/layout-component">Screenshot of /learn/basics/assets-metadata-css/layout-component</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/FxtUdzLxkVg___oyMoGqNAAc6PU_.png">(view full size)</a> |<a href="https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/assets-metadata-css/metadata">Screenshot of /learn/basics/assets-metadata-css/metadata</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/uckEke13QXwBYwgkciz_Cl56RVo_.png">(view full size)</a> |



| [**…getstaticprops-details**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/data-fetching/getstaticprops-details) | [**…fetching/pre-rendering**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/data-fetching/pre-rendering) |
|:-:|:-:|
|<a href="https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/data-fetching/getstaticprops-details">Screenshot of /learn/basics/data-fetching/getstaticprops-details</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/XrA7NyrYKqgkAhFJR6ojFNlzVXo_.png">(view full size)</a> |<a href="https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/data-fetching/pre-rendering">Screenshot of /learn/basics/data-fetching/pre-rendering</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/a0psarFPxvKoGXSkxMhwMhUxaWE_.png">(view full size)</a> |



And 3 other routes:
- [**/learn/basics/data-fetching/request-time**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/data-fetching/request-time)
- [**/learn/basics/dynamic-routes/dynamic-routes-details**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/dynamic-routes/dynamic-routes-details)
- [**/learn/basics/dynamic-routes/render-markdown**](https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh/learn/basics/dynamic-routes/render-markdown)



Commit d866c88e66d284c8d787d3a10195bd7806e10eb4 (https://next-site-git-fork-nminhnguyen-suggestions.zeit.now.sh).

File: pages/learn/basics/dynamic-routes/dynamic-routes-details.mdx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const meta = {
   subtitle: 'Dynamic Routes Details'
 }
 
-You can get in-depth information about dynamic routes on our documentation:
+You can get in-depth information about dynamic routes in our documentation:
```
Comment (chibicode) [inline pages/learn/basics/dynamic-routes/dynamic-routes-details.mdx:12]: I think "on" might be ok when referring to the documentation  🤔

Screen Shot 2020-04-27 at 12 04 57 AM

Screen Shot 2020-04-27 at 12 04 52 AM

---

Screen Shot 2020-04-27 at 12 06 12 AM

Screen Shot 2020-04-27 at 12 07 18 AM

Comment (NMinhNguyen) [reply pages/learn/basics/dynamic-routes/dynamic-routes-details.mdx:12]: I don't have very strong opinions on this one, but e.g. https://reactjs.org/blog/2013/10/16/react-v0.5.0.html contains "in the docs", https://reactjs.org/docs/getting-started.html#something-missing contains "in the documentation" (more results from https://reactjs.org [here](https://www.google.com/search?ei=eoamXpfvGeKChbIPpI2jiA8&q=site%3Areactjs.org+%22in+the+documentation%22&oq=site%3Areactjs.org+%22in+the+documentation%22&gs_lcp=CgZwc3ktYWIQAzoECAAQR1DOH1jjIGC7IWgAcAJ4AIABIIgBXZIBATOYAQCgAQGqAQdnd3Mtd2l6&sclient=psy-ab&ved=0ahUKEwiXlObLh4jpAhViQUEAHaTGCPEQ4dUDCAw&uact=5)), https://nextjs.org/learn/basics/data-fetching/request-time already contains "in our documentation"

Comment (chibicode) [reply pages/learn/basics/dynamic-routes/dynamic-routes-details.mdx:12]: @NMinhNguyen got it, thanks!

**Pull request merged**